projects/desktop/src/assets/locales/EN.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Add trailing coma in tslint.json ?

Reply:
Trailing comma is not authorized in json files

Reply:
Okay, don't clash me